prerequisites.md

```diff
@@ -91,30 +108,39 @@ Where the -i flag stands for `identity`, and `ubuntu` is the default username.
     - If the portable version is used, you will need to start the bash from the executable found in the unpacked folder.
 4. If needed, navigate to a preferred folder. (use `pwd` to figure out where you are, use `cd` to navigate)
 
-## Generating a keypair in Git Bash
 <a name="gitbash"></a>
+
+## Generating a keypair in Git Bash
+
 1. In Git Bash, type `ssh-keygen -o`, and follow the instructions.
```

Comment:
wouldn't a direct `ssh-keygen -t rsa -b 2048 -a 16` work?

Reply:
It should work, but every flag has the potential for a long-winded discussion. Also, doesn't ssh-keygen default to rsa and 2048 bits anyway? What is the added value of these flags?

Reply:
Alright, perhaps a training is not the time and place to have that discussion. I'm not sure if openssh defaults to those values on all platforms though.
Another reason I made the comment is that the command for other setups does have (some of) the explicit flags.

Reply:
ssh-keygen does not behave in the exact same way in all systems, and this would require some attention. However, we're thinking of generating the keys ourselves, and sending the private keys to the participants, due to putty-related nonsense and the subsequent delays during the training. Due to this, I'd say leave the commands as they are right now; we'll first investigate/test if producing the keys ourselves is indeed an improvement.